MPCAutofill/cardpicker/tags.py

```diff
@@ -1,31 +1,59 @@
 """
 Tags for cards.
 """
-from __future__ import annotations
 
+import functools
 import re
+from typing import Optional
 
-from aenum import MultiValueEnum
+from aenum import MultiValueEnum, extend_enum
+
+from cardpicker import models
 
 
 class Tag(str, MultiValueEnum):
-    ALT_ART = "ALT ART", "ALTART", "ALT"
-    EXTENDED = "EXTENDED"
-    FULL_ART = "FULL ART", "FULLART", "FULL"
-    NSFW = "NSFW"
+    ALT_ART = "Alt Art", "Alternative Art", "Alternate Art", "Alt"
+    EXTENDED = "Extended", "EXTENDED Art"
+    FULL_ART = "Full Art", "FULL ART", "Fullart", "Full"
+    NSFW = "NSFW", "Nsfw"
```

Comment:
Some are still uppercase

Reply:
i discovered that if you read the `.value` of a multi-value enum, the string you receive is the first defined value of the enum. in this case, i want the `NSFW` tag to be displayed as `NSFW` to the user, but the title-case transformation you commented on further down in this file means that it will match to the `NSFW` enum item with the string `Nsfw`.

Reply:
I understand for NSFW display, but for extended and full, the uppercase values `FUL L ART` and `EXTENDED Art` are not the first ones so they will not be used. And not sure if `Extended Art` will currently match.

Reply:
ah yep, i forgot about those - will fix